src/static/riot/queues/list.tag

```diff
@@ -0,0 +1,282 @@
+                        remoteValues: 'results',
+                        name: 'username',
+                        value: 'id',
+                    },
+                    cache: false,
+                    maxResults: 5,
+                })
+            ;
+        })
+
+        self.switch_to_form = () => {
+            window.location = '/queues/form/'
+        }
+
+        self.update_queues = function (filters) {
+            filters = filters || {}
+            let show_public_queues = $(self.refs.public).prop('checked')
+            if (show_public_queues) {
+                filters.public = true
+            }
+            CODALAB.api.get_queues(filters)
+                .done(function (data) {
+                    self.queues = _.orderBy(data.results, queue => !queue.is_owner);
+                    self.pagination = {
+                        "count": data.count,
+                        "next": data.next,
+                        "previous": data.previous
+                    }
+                    self.update()
+                })
+                .fail(function (response) {
+                    toastr.error("Could not load tasks")
+                })
+        }
+
+        /*---------------------------------------------------------------------
+        Modal Methods
+        ----------------------------------------------------------------------*/
+        self.show_modal = (queue) => {
+            if (queue !== undefined && queue !== null) {
+                self.set_selected_queue(queue)
+            }
+            $(self.refs.modal).modal('show')
+        }
+
+        self.close_modal = () => {
+            $(self.refs.modal).modal('hide')
+            self.clear_form()
+        }
+
+        self.clear_form = () => {
+            $('.collab-search').dropdown('clear')
+            self.refs.queue_name.value = ''
+            self.selected_queue = {}
+            self.refs.queue_public.checked = false
+        }
+
+        self.set_selected_queue = function (queue) {
+            self.selected_queue = queue
+            $('.collab-search')
+                .dropdown('setup menu', {values: queue.organizers})
+                .dropdown('set selected',  _.map(queue.organizers, o => o.username))
+
+            if (queue.is_public) {
+                self.refs.queue_public.checked = true
+            }
+        }
+
+        self.handle_queue = function () {
+            let data = {
+                name: self.refs.queue_name.value,
+                is_public: self.refs.queue_public.checked,
+                organizers: _.compact($('.collab-search').dropdown('get value').split(','))
+            }
+            let endpoint = !_.isEmpty(self.selected_queue)
+                ? CODALAB.api.update_queue
+                : CODALAB.api.create_queue
+            endpoint(data, _.get(self, 'selected_queue.id'))
+                .done(function (response) {
+                    toastr.success("Succesfully updated/made queue!")
```

Comment:
This success message wording should either be changed to be agnostic of whether update or create was called. Or we should determine which was done and display the correct message.

Reply:
Talked to you about this, should be addressed